src/components/ec-full-screen-overlay/ec-full-screen-overlay.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+  <transition name="ec-full-screen-overlay__fade">
+    <div
+      v-if="show"
+      class="ec-full-screen-overlay"
+      data-test="ec-full-screen-overlay"
+    >
+      <div
+        ref="overlayContent"
+        class="ec-full-screen-overlay__content"
+        data-test="ec-full-screen-overlay__content"
+      >
+        <header
+          class="ec-full-screen-overlay__header"
+          data-test="ec-full-screen-overlay__header"
+        >
+          <h1
+            v-if="title"
+            data-test="ec-full-screen-overlay__title"
+            class="ec-full-screen-overlay__title"
+          >
+            {{ title }}
+          </h1>
+
+          <button
+            class="ec-full-screen-overlay__close-icon-btn"
+            data-test="ec-full-screen-overlay__close-icon-btn"
+            @click="emit('close-overlay')"
```

Comment:
Silly thing but can we please make this `@click="emit('close')"`

We tend to have it like that everywhere else. We do not repeat the name of the component in the event.
So we have simple event names like `close`, `change`, `hide` etc.

Thank you